skyd/api.go

```diff
@@ -68,6 +71,38 @@ func NewAPI(nginxHost string, nginxPort int, skydHost, skydPassword string, skyd
 	}, nil
 }
 
+// Blocklist returns a list of hashes that make up the blocklist.
+func (api *api) Blocklist() ([]crypto.Hash, error) {
+	url := fmt.Sprintf("http://%s:%d/skynet/blocklist?timeout=%s", api.staticSkydHost, api.staticSkydPort, skydTimeout)
```

Comment:
This seems like it should use our client library because it's just a lightweight reimplementation of it which doesn't check the status code and uses a bunch of hardcoded values.

Reply:
is this F/U material? definitely a good suggestion though

Reply:
done